deb/Makefile

```diff
@@ -1,7 +1,7 @@
 all: deb-build
 
 clean:
-	rm -rf output brave-keyring/etc/apt/trusted.gpg.d/* *.asc
+	rm -rf output brave-keyring/etc/apt/trusted.gpg.d/* brave-keyring/usr
```

Comment:
Nit: since we're doing an explicit `mkdir -p`, this could just be `.../trusted.gpg.d` without the `/*`.